service/migrations/03_identity_read_models.sql

```diff
@@ -0,0 +1,136 @@
+-- Core identity read models
+CREATE TABLE IF NOT EXISTS accounts (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    username TEXT NOT NULL UNIQUE,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    root_kid TEXT NOT NULL,
+    root_pubkey TEXT NOT NULL,
+    tier TEXT NOT NULL DEFAULT 'anonymous' CHECK (tier IN ('anonymous', 'verified', 'bonded', 'vouched')),
```

Comment:
I'm guessing these 3 can be removed without breaking anything. I don't think this is how we want to support these features and we're not ready for them yet.